app/state_test.go

```diff
@@ -494,7 +499,7 @@ func TestStateInvetoryUpdate(t *testing.T) {
 	s, _ := ius.Handle(ctx, &stateTestController{
 		inventoryErr: errors.New("some err"),
 	})
-	assert.IsType(t, &checkWaitState{}, s)
+	assert.IsType(t, &inventoryUpdateRetry{}, s)
```

Comment:
Would be great to test the new wait state as well, which should get the coverage up.

Reply:
added! https://github.com/mendersoftware/mender/pull/943/commits/383fca9e42d0f52253be1f872e48554e6585ce97#diff-ef52b340986b4690b1936c29f3f7e22bd4df65857dad63ffd279e72b40838d8fR515

and in passing also removed the unneeded argument here: https://github.com/mendersoftware/mender/compare/d21fe3d1b38b29df6d91796a3865bd3e3cac47ab..7d9bae4484013fc2c7f1baaa60a02b5d1c772504#diff-0b03a1a30843453d28f38533c95445ff2cd5009a25d18c640ef66426efc044e8R1114